.github/workflows/docker-dev.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Dev Docker Image
+
+concurrency:
+  group: ${{ github.workflow }}-${{ inputs.branch || inputs.commit }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Name of the branch to build. Is prioritised over the commit input."
+        type: string
+        default: ""
+      commit:
+        description: "Commit to build. One of branch or commit is required. If both are provided, branch is used."
+        type: string
+        default: ""
+
+jobs:
+  build-and-push-docker-image:
+    name: Build image and push to GHCR
+    runs-on: ubuntu-24.04
+    # Run the job only if the PR is not a draft.
+    # This is done to limit the runner cost.
+    if: inputs.branch != '' || inputs.commit != ''
+    timeout-minutes: 30
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v5
+        with:
+          ref: ${{ inputs.branch || inputs.commit }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Github Packages
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Docker Meta fat for PRs only
+        id: metafat
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/chainsafe/forest
+          flavor: |
+            latest=false
+          tags: |
+            type=raw,value={{date 'YYYY-MM-DD'}}-{{sha}}-fat
+
+      - name: Build fat image and push to GitHub Container Registry
+        uses: docker/build-push-action@v6
+        with:
+          file: ./Dockerfile
+          tags: ${{ steps.metafat.outputs.tags }}
+          labels: ${{ steps.metafat.outputs.labels }}
+          push: true
+          # Compile Docker image only for AMD64 for a regular PR to save some CI time.
```

Comment:
This comment is incorrect (this workflow is NOT part of a CI). Leftover from copypasta?